themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +34,34 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group']): ?>
+                    <?php if (isset($this->config->AuthorResultListInfo->ResultListPage) && ($this->config->AuthorResultListInfo->ResultListPage == 'Short')): ?>
```

Comment:
What if, instead of having a nested if here, you made the check above:

`elseif ('Au' == $item['Group'] && strtolower($this->config->AuthorResultListInfo->ResultListPage ?? 'Long') == 'short'):`

That simplifies the isset, incorporates my case-insensitive suggestion from above, and allows you to utilize the existing else clause below instead of having to make a second nested copy of the logic.

Reply:
thanks for the suggestion and explanation @demiankatz , really appreciate it!

I've included that in my latest follow-up